providers/base/bin/zapper_keyboard_test.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+"""
+This program tests whether the system responds correctly to an external
+USB keyboard.
+
+To run the test you need Zapper board connected and set up.
+"""
+import os
+import struct
+import sys
+import threading
+from enum import Enum
+
+from checkbox_support.scripts.zapper_proxy import zapper_run  # noqa: E402
+
+ZAPPER_KBD = "usb-Canonical_Zapper_main_board_123456-event-kbd"
```

Comment:
Will this be used by some external entity?
If not, maybe it's better to keep it local to a place where it's used (as there's only one place it's used).

Reply:
done in https://github.com/canonical/checkbox/pull/707/commits/600ea83d429edd36a78330f525d340610084e5d6